mysite/urls.py

```diff
@@ -1,3 +1,9 @@
+"""Store all urls used in project.
+
+- admin/ - admin control panel;
```

Comment:
This comment is not really necessary, since the code below is already self-descriptive. Also, you may forget to update it as fast as you update the routes themselves.